mypy/stubutil.py

```diff
@@ -90,7 +90,8 @@ def is_c_module(module: ModuleType) -> bool:
     return '__file__' not in module.__dict__ or module.__dict__['__file__'].endswith('.so')
 
 
-def write_header(file: IO[str], module_name: str, pyversion: Tuple[int, int] = (3, 5)) -> None:
+def write_header(file: IO[str], module_name: str = None,
```

Comment:
I prefer Optional[str], EIBTI. (I somewhat regret the rule in PEP 484 about this, or at least I don't enjoy relying on it.)